File: src/main/java/com/server/computer_science/question/common/dto/request/RequestChangeContentDto.java
package com.server.computer_science.question.common.dto.request;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@NoArgsConstructor
@AllArgsConstructor
@Builder
@Getter
public class RequestChangeContentDto {
	private String content;

	public static RequestChangeContentDto forTest() {
		return RequestChangeContentDto.builder()
			.content("test")
			.build();
	}
}

File: src/main/java/com/server/computerscience/question/common/dto/request/RequestChangeContentDto.kt
package com.server.computerscience.question.common.dto.request


data class RequestChangeContentDto(
    val content: String? = null
) {
    companion object {
        @JvmStatic
        fun forTest(): RequestChangeContentDto {
            return RequestChangeContentDto(
                content = "test"
            )
        }
    }
}
